tests/reporters/test_custom_x_reporter.py

```diff
@@ -0,0 +1,344 @@
+"""
+Tests covering the IntervalReporter class from
```

Comment:
Update all your docstrings in this file since they're all from `IntervalReporter`